youtube_dl/extractor/piksel.py

```diff
@@ -113,11 +110,18 @@ def _real_extract(self, url):
             })
         self._sort_formats(formats)
 
+        subtitles = {}
+        for caption in video_data.get('captions', []):
+            if 'url' in caption:
```

Comment:
will allow the url value to be `None`, instead check for the validity of the value.
squash the commits after this change.

Reply:
yes. will do.

Reply:
instead of repeating access to the `url` key(`caption.get('url')`), put the value in variable and reuse it, squash commit to a single commit(the first one) so it can be directly merged.

Reply:
done